### PR TITLE
Update README with Integration Domain support

### PR DESCRIPTION
**Pull request by justin-stephenson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,13 @@ cd $IPA_TUURA/src/install
 python prepare_sssd.py
 ```
 
+### Keycloak integration domain provisioning
+
+Alternatively, auto-enroll the host by providing the completing the required
```
Comment (spoore1) [inline README.md:33]: I think you should drop "the completing" here.  Looks like a typo?

**Review by spoore1 — COMMENTED** [review]
I'm curious if this screen shot will become out of date with keycloak updates.   Should there be a note that the interface or keycloak form may change but the fields should remain valid?  Or just wait and change the image if/when we need to?

File: README.md (hunk 1 above)
Comment (justin-stephenson) [reply README.md]: Fixed, thanks.

**Comment by justin-stephenson** [comment]
> I'm curious if this screen shot will become out of date with keycloak updates. Should there be a note that the interface or keycloak form may change but the fields should remain valid? Or just wait and change the image if/when we need to?

Thank you for the review Scott.

I would prefer to change it if/when it is needed, if minor changes are made to the form it probably does not need to be explicitly stated.

**Comment by spoore1** [comment]
> > I'm curious if this screen shot will become out of date with keycloak updates. Should there be a note that the interface or keycloak form may change but the fields should remain valid? Or just wait and change the image if/when we need to?
> 
> Thank you for the review Scott.
> 
> I would prefer to change it if/when it is needed, if minor changes are made to the form it probably does not need to be explicitly stated.

Sounds good to me.

**Review by spoore1 — APPROVED** [review]
LGTM

**Review by f-trivino — APPROVED** [review]

**Pull request merged**